api/helpers/data/tagkeys.json

```diff
@@ -0,0 +1,49 @@
+[
```

Comment:
i have a list of all the keys for all the select fields that are being used on the front end. i wonder if we can share that same file as the source of truth for keys. https://github.com/participedia/api/blob/master/api/helpers/shared-field-options.js#L303

Reply:
That would be excellent.